src/views/tickets/orders-rsvp.php

```diff
@@ -17,57 +17,76 @@
 $post      = get_post( $post_id );
 $post_type = get_post_type_object( $post->post_type );
 $user_id   = get_current_user_id();
+$user_info = get_userdata( $user_id );
 $attendees = $view->get_event_rsvp_attendees( $post_id, $user_id );
+$first_attendee = $attendees[0];
```

Comment:
This should be relocated below the `if ( ! $view->has_rsvp_attendees( $post_id, $user_id ) ) {` block so we avoid notices when there aren't any attendees.